test/subvol_test.rb

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env rspec
+
+require_relative "spec_helper"
+require "storage/subvol"
+require "pp"
+Yast.import "Arch"
+
+
+describe Yast::StorageClass::Subvol do
+
+  context "#new" do
```

Comment:
I would prefer a different approach focused in each method:

```ruby
describe "#no_cow?" do
  subject { Subvol.new("/var/lib") }

  context "by default" do
    it "returns true" do
      expect(subject.no_cow?).to eq(true)
    end
  end

  context "when is not a nocow volume" do
    subject { Subvol.new("/var/lib/pgsql", noco) }

    it "returns false" do
      expect(subject.no_cow?).to eq(false)
    end
  end

  # ...
end
```

But it's fine for me as it.